app/assets/stylesheets/header.scss

```diff
@@ -1,8 +1,16 @@
 .header{
-  border-bottom:solid 2px #5DC9C9;
+  position: fixed;
+  left: 0;
+  top: 0;
+  width: 100%;
+  border-bottom:  2px solid #dcdcdc;
```

Comment:
Color literals like `#dcdcdc` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Colon after property should be followed by one space